tests/auth_tests/test_hashers.py

```diff
@@ -520,6 +521,22 @@ def test_encode_invalid_salt(self):
                     with self.assertRaisesMessage(ValueError, msg):
                         hasher.encode("password", salt)
 
+    def test_password_bytes(self):
+        hasher_classes = [
+            Argon2PasswordHasher,
+            BCryptPasswordHasher,
+            BCryptSHA256PasswordHasher,
+            MD5PasswordHasher,
+            PBKDF2PasswordHasher,
+            PBKDF2SHA1PasswordHasher,
+            ScryptPasswordHasher,
+        ]
+        for hasher_class in hasher_classes:
+            hasher = hasher_class()
+            with self.subTest(hasher_class.__name__):
+                encoded = hasher.encode(b"password", hasher.salt())
+                self.assertTrue(hasher.verify(b"password", encoded))
```

Comment:
Nit: Django prefers using `assertIs` for boolean checks like this.
```suggestion
                self.assertIs(hasher.verify(b"password", encoded), True)
```

Reply:
Thanks for pointing out, I've applied your suggestion.